modules/social_features/social_event/modules/social_event_addtocal/src/Plugin/SocialAddToCalendarBase.php

```diff
@@ -106,17 +108,17 @@ public function getEventDates(NodeInterface $node) {
     $date_time = [];
 
     // Set formats for event dates.
-    $format = $this->pluginDefinition['dateFormat'];
+    $format = $this->pluginDefinition['dateFormat'] ?? 'Ymd\THis';
```

Comment:
I believe our code are safe to these data never arrived empty here, because never had an error here, but, I thought in a default value to simplify the solution. I opened a discuss about default values here: https://opensocial.slack.com/archives/G01CXDZ0FCG/p1710789447543369 and I got from here: html/profiles/contrib/social/modules/social_features/social_event/modules/social_event_addtocal/src/Annotation/SocialAddToCalendar.php